src/data/finite/card.lean

```diff
@@ -139,6 +142,27 @@ by { haveI := fintype.of_finite α, haveI := fintype.of_finite β, simp }
 
 end finite
 
+namespace set
+
+lemma card_union_le {s t : set α} : nat.card ↥(s ∪ t) ≤ nat.card s + nat.card t :=
+begin
+  casesI _root_.finite_or_infinite ↥(s ∪ t) with h h,
+  { rw [finite_coe_iff, finite_union, ←finite_coe_iff, ←finite_coe_iff] at h,
+    casesI h,
+    rw [←cardinal.nat_cast_le, nat.cast_add,
+        finite.cast_card_eq_mk, finite.cast_card_eq_mk, finite.cast_card_eq_mk],
+    exact cardinal.mk_union_le s t },
+  { exact nat.card_eq_zero_of_infinite.trans_le (zero_le _) },
+end
+
+lemma card_image_le {s : set α} [finite s] {f : α → β} : nat.card (f '' s) ≤ nat.card s :=
+finite.card_le_of_surjective _ surjective_onto_image
+
+lemma card_range_le [finite α] {f : α → β} : nat.card (set.range f) ≤ nat.card α :=
+finite.card_le_of_surjective _ surjective_onto_range
```

Comment:
I think these two lemmas belong to the `finite` namespace since they're similar to the existing lemmas below them.

Also these three lemmas are inequalities, so I think the arguments should be made explicit, like the existing lemmas below.